README.md

```diff
@@ -35,7 +35,7 @@ ap object, options = {}
 Default options:
 
 ```ruby
-indent:        4,      # Number of spaces for indenting.
+indent:        2,      # Number of spaces for indenting.
```

Comment:
This doesn't look like it matches `master` branch https://github.com/amazing-print/amazing_print/blob/master/lib/amazing_print/inspector.rb#L18

Reply:
Looks like the change was reverted in 707e5e66a1e4eb29b7edfe9f9e43362b7ec1e03f.

I don't know why — using the same indent as the rest of the Ruby world seems to make sense to me. But I guess this PR is irrelevant as a result…